Okta.AspNet.Test/OktaHttpMessageHandlerShould.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NSubstitute;
+using Okta.AspNet.Abstractions;
+using Xunit;
+
+namespace Okta.AspNet.Test
+{
+    public class OktaHttpMessageHandlerShould
+    {
+        [Fact]
+        public async Task SetInnerHandlerWebProxy()
+        {
+            await Task.Run(() =>
```

Comment:
Curious, why do we need this test to be async?

Reply:
I believe I did this in response to style warning.

Reply:
Probably StyleCop complained because this is ` public async Task `. If we change this to a simple `void` method, then `Task.Run()` shouldn't be necessary. This is not a blocker at all, but I suggest we simplify the test by removing Task.Run.